src/Symfony/Component/Translation/Tests/TranslatorTest.php

```diff
@@ -499,8 +499,10 @@ public function testTransChoiceFallbackWithNoTranslation()
     /**
      * @dataProvider dataProviderGetMessages
      */
-    public function testGetMessages($resources, $locale, $expected)
+    public function testLegacyGetMessages($resources, $locale, $expected)
     {
+        $this->iniSet('error_reporting', -1 & ~E_USER_DEPRECATED);
```

Comment:
is this still  required? I've seen tests without it and sometimes a `@group legacy` annotation. What exactly is the workflow?